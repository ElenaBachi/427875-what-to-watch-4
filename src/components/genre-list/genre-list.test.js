import React from "react";
import renderer from "react-test-renderer";
import GenreList from "./genre-list.jsx";

const mock = {
  genres: [`All genres`, `Comedies`, `Crime`, `Documentary`, `Dramas`, `Horror`]
};

it(`GenreList should render correctly`, () => {
  const tree = renderer.create(
      <GenreList
        genres={mock.genres}
        onFilterCLick={() => {}}
        currentGenre={mock.genres[0]}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
