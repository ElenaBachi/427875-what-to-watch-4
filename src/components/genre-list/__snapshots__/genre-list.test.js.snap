// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenreList should render correctly 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item catalog__genres-item--active"
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      All genres
    </a>
  </li>
  <li
    className="catalog__genres-item "
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      genre1
    </a>
  </li>
  <li
    className="catalog__genres-item "
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      genre2
    </a>
  </li>
  <li
    className="catalog__genres-item "
    onClick={[Function]}
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      genre3
    </a>
  </li>
</ul>
`;
